modules/cloud-source-ocm.adoc

```diff
@@ -0,0 +1,24 @@
+// Module included in the following assemblies:
+//
+// * integration/integrate-with-ocm.adoc
+:_mod-docs-content-type: PROCEDURE
+[id="cloud-source-ocm_{context}"]
+= Configuring OpenShift Cluster Manager integration
+
+To discover cluster assets from OpenShift Cluster Manager, create a new integration in {product-title}.
+
+.Prerequisites
+* A Red Hat account.
+* An link:https://console.redhat.com/openshift/token[OpenShift Cluster Manager API token].
+
+.Procedure
+. On the {product-title-short} portal, navigate to *Platform Configuration* -> *Integrations*.
+. Scroll down to the *Cloud source integrations* section and select *OpenShift Cluster Manager*.
+. Click *New Integration* (*`add`* icon).
```

Comment:
Same feedback about removing literals from UI terms (if you're reading this on GitHub diff and see this comment first, [here is where](https://github.com/openshift/openshift-docs/pull/71977/files#diff-88eaebaf9aef798367ffe87934b249217de15c308da162f4723194b00630f448R17) I elaborate):
```suggestion
. Click *New Integration* (*add* icon).
```

Reply:
see above, removed the bracket part.